.github/workflows/ci.yml

```diff
@@ -23,9 +23,9 @@ jobs:
                 path: |
                   ~/.gradle/caches
                   ~/.gradle/wrapper
-                key: ${{ runner.os }}-gradle-false-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+                key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
                 restore-keys: |
-                  ${{ runner.os }}-gradle-false-
+                  ${{ runner.os }}-gradle-
```

Comment:
Remove this line, the next one is the same.

Reply:
Done